bootloader/grub.go

```diff
@@ -463,6 +463,20 @@ func staticCommandLineForGrubAssetEdition(asset string, edition uint) string {
 	return string(cmdline)
 }
 
+var (
+	recoveryModeTrustedAssets = []string{
```

Comment:
the package has all the bootloaders, better for this vars to start with "grub..."